### DISCUSSION TITLE
Rich Presence buttons

### DISCUSSION THREAD

**Discussion by Evanlau1798** [LennyPhoenix/py-discord-sdk#15] [Q&A]
Excuse me
How can I set Rich Presence buttons by using py-discord-sdk?
I cant find any related offical docs and wiki.

**Comment by LennyPhoenix** [comment]
It's not currently possible to set Rich Presence buttons up using py-discord-sdk, as the interface is not exposed by the Discord SDK.
However, it is possible to use the IPC protocol directly which will allow you to do this.

**Reply by Evanlau1798** [reply]
Got it,thanks for your advice.
